### DISCUSSION TITLE
6-2 Hawaii right?

### DISCUSSION THREAD

**Discussion by aaronbirr** [ua-gist604b-f24/syllabus#32] [General]
The assignment says to upload a screen cap of reykjavik, but it should be Hawaii, correct? 

 
![Screenshot 2024-11-23 190900](https://github.com/user-attachments/assets/e8cd2c49-3235-4cbb-8036-7af33ac11949)

**Comment by mcguckinj** [comment]
I think you're right, it's supposed to be Hawaii.
